src/main/java/uk/gov/companieshouse/overseasentitiesapi/validation/EntityDtoValidator.java

```diff
@@ -132,6 +141,12 @@ private Errors validatePublicRegisterNameIsNotSupplied(String publicRegisterName
         return errors;
     }
 
+    private Errors validatePublicRegisterJurisdictionIsNotSupplied(String publicRegisterJurisdiction, Errors errors, String loggingContext) {
```

Comment:
i would move the position of this jurisdiction validator up in the class , so that it is next to the other jurisdiction validator, then the registration validators will also be next to each other. Just makes it a bit easier to find things in the class.

Reply:
I'm inclined to keep it as it is we have the three validators then the three not supplied validators seems logical either way.

Reply:
ok